recipes/myproxyclient/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "MyProxyClient" %}
+{% set version = "1.4.3" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "5b7ad3cc3c3cd71595a9b0278a51c138699e229d47a41f2e2add72fbc12f58cd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  entry_points:
+    - myproxyclient = myproxy.script:main
+  skip: True  # [py3k]
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyopenssl
+    - certifi
+
+  run:
+    - python
+    - pyopenssl
```

Comment:
@jarifibrahim Seems odd, I did not need `certifi` in the original recipe to get the build to work on my own machine.